x-pack/test/api_integration/apis/siem/saved_objects/timeline.ts

```diff
@@ -296,18 +295,15 @@ const timelinePersistenceTests: KbnTestProvider = ({ getService }) => {
       });
     });
   });
-};
-
-// eslint-disable-next-line import/no-default-export
-export default timelinePersistenceTests;
+}
 
 const omitTypename = (key: string, value: keyof TimelineResult) =>
   key === '__typename' ? undefined : value;
 
 const omitTypenameInTimeline = (timeline: TimelineResult) =>
   JSON.parse(JSON.stringify(timeline), omitTypename);
 
-const createBasicTimeline = async (client: ApolloClient<InMemoryCache>, titleToSaved: string) =>
+const createBasicTimeline = async (client: ApolloClient<any>, titleToSaved: string) =>
```

Comment:
Can we not use `ApolloCient<InMemoryCache>` anymore?

Reply:
Because the function that makes it actually returns an `ApolloClient<NormaliedCacheStore>` or something like that, and I wasn't sure how to get that type, and this function doesn't actually cache what the store/cache type is.